package.json

```diff
@@ -7,9 +7,11 @@
     "lint": "eslint \"src/**/*.js\" \"test/**/*.js\" \"examples/**/*.js\"",
     "doc": "jsdoc -t node_modules/docdash --readme JSDOC.md -c jsdoc-config.json",
     "doclint": "npm run doc -- -t templates/silent",
-    "test": "npm run lint -- --max-warnings=0 && npm run build && npm run test-examples && npm run test-unit",
+    "test": "npm run lint -- --max-warnings=0 && npm run build && npm run test-examples && npm run test-with-coverage",
     "test-unit": "cross-env BABEL_DISABLE_CACHE=1 mocha --require babel-core/register test/*unit_test.js",
     "test-examples": "mocha -t 300000 --recursive test/examples",
+    "test-with-coverage": "nyc -n src -r html cross-env BABEL_DISABLE_CACHE=1 mocha --require babel-core/register test/*unit_test.js",
+    "deploy-coverage": "nyc report --reporter=text-lcov | coveralls",
```

Comment:
I'm a bit concerned that this uses a tool that's not listed as a dependency (too bad there's no "optional dev dependencies").

Could maybe the Travis build be rewritten to use `npx nyc report --reporter=txt-lcov | npx coveralls` instead of `npm run deploy-coverage`?

Reply:
ok, I've remove the line from package.json, and move it to .travis.yml